### PR TITLE
show "manual login" button only if login failed due to captcha

### PR DESCRIPTION
**Pull request by ztNFny**
show "manual login" button only if login failed due to captcha
fixes https://github.com/cgeo/cgeo/issues/16331

make GC login errors localizable
fixes https://github.com/cgeo/cgeo/issues/16337

### CODE REVIEW

File: main/src/main/java/cgeo/geocaching/MainActivity.java [hunk 1]
```diff
@@ -155,7 +155,7 @@ private void fillView(final View connectorInfo, final ILogin conn) {
                             connectorStatus.setOnClickListener(connectorConfig);
 
                             final Button manualLogin = connectorInfo.findViewById(R.id.manual_login);
-                            manualLogin.setVisibility(!conn.isLoggedIn() && !isLoggingIn && !isLoggingOk && conn.supportsManualLogin() && Settings.getGcCredentials().isValid() ? View.VISIBLE : View.GONE);
+                            manualLogin.setVisibility(connInfo.toString().contains("Failed to log in to geocaching.com due to captcha required") ? View.VISIBLE : View.GONE);
```
Comment (eddiemuc) [inline main/src/main/java/cgeo/geocaching/MainActivity.java:158]: Does this work for non-english installations?

Comment (eddiemuc) [reply main/src/main/java/cgeo/geocaching/MainActivity.java:158]: Never mind, I just saw your new issue about error messages being untranslated :-)

**Comment by eddiemuc** [comment]
@ztNFny can you rebase this onto release?

**Comment by ztNFny** [comment]
> @ztNFny can you rebase this onto release?

done

**Comment by ztNFny** [comment]
I added the localization of error messages as well

**Pull request merged**